src/routes/productsRout.ts

```diff
@@ -0,0 +1,21 @@
+import express, { Router } from "express";
+import {
+  getProducts,
+  getProduct,
+  getProductViaCategory,
+  setProduct,
+  updateProduct,
+  deleteProduct,
+} from "../controllers/productsController";
+
+const productsRout = Router();
+productsRout.use(express.json());
+
+productsRout.get("/all", getProducts);
```

Comment:
you can remove 'all', since you have a /products/:id for singe one and for all you should have a /products. It will keep uniform interface for all /products endpoints.